finality-aleph/src/justification.rs

```diff
@@ -245,3 +245,23 @@ where
         ))
     }
 }
+
+/// Old format of justifications, needed for backwards compatibility.
+#[derive(Clone, Encode, Decode, Debug)]
+pub(crate) struct AlephJustificationV1 {
+    pub(crate) signature: SignatureSet<SignatureV1>,
+}
+
+impl From<AlephJustificationV1> for AlephJustification {
+    fn from(just_v1: AlephJustificationV1) -> AlephJustification {
+        let mut just_drop_id: SignatureSet<Signature> =
+            SignatureSet::with_size(just_v1.signature.size());
+        just_v1
+            .signature
+            .into_iter()
+            .for_each(|(id, sgn)| just_drop_id.insert(id, sgn.into()));
+        AlephJustification {
+            signature: just_drop_id,
+        }
+    }
+}
```

Comment:
maybe smth like
```suggestion
impl From<AlephJustificationV1> for AlephJustification {
    fn from(just_v1: AlephJustificationV1) -> AlephJustification {
        let size = just_v1.signature.size();
        let just_drop_id: SignatureSet<Signature> = just_v1
            .signature
            .into_iter()
            .fold(SignatureSet::with_size(size), |sig_set, (id, sgn)| {
                sig_set.add_signature(&sgn.into(), id)
            });
        AlephJustification {
            signature: just_drop_id,
        }
    }
}
```

(just wondering, surely non-blocker)

Reply:
Addressed

Reply:
thanks a lot!